docs/host_dev.md

```diff
@@ -0,0 +1,153 @@
+# uFork Host Device
+
+The **Host Device** extends the set of built-in devices with **dynamic**
+devices. Dynamic devices can be installed without recompiling the uFork WASM
+core, and even whilst the core is running.
```

Comment:
I think this is not quite correct. Dynamic devices can be installed/disposed anytime the core is **idle**, but technically not _while the core is running_.

Reply:
Perhaps the term _suspended_ is better than _idle_? The point is, it has to be between event/instruction dispatch cycles. Essentially, anytime you could call the re-entrant `h_` methods.